src/datepicker/datepicker.ts

```diff
@@ -133,6 +136,13 @@ export class NgbDatepicker implements OnDestroy,
    */
   @Input() firstDayOfWeek: number;
 
+  /**
+   * Reference for the custom template for the footer
+   *
+   * @since 3.3.0
+   */
+  @Input() footerTemplate: TemplateRef<FooterTemplateContext>;
```

Comment:
`TemplateRef<{}>` I guess...

Reply:
To be consistent with other places `TemplateRef<any>` was used.